common/djangoapps/student/urls.py

```diff
@@ -0,0 +1,64 @@
+"""
+URLs for student app
+"""
+
+from django.conf import settings
+from django.conf.urls import patterns, url
+
+from student.views import LogoutView
+
+urlpatterns = (
+    'student.views',
+
+    url(r'^logout$', LogoutView.as_view(), name='logout'),
+
+    # TODO: standardize login
+
+    # login endpoint used by cms.
+    url(r'^login_post$', 'login_user', name='login_post'),
+    # login endpoints used by lms.
+    url(r'^login_ajax$', 'login_user', name="login"),
+    url(r'^login_ajax/(?P<error>[^/]*)$', 'login_user'),
+
+    url(r'^email_confirm/(?P<key>[^/]*)$', 'confirm_email_change'),
+
+    url(r'^create_account$', 'create_account', name='create_account'),
+    url(r'^activate/(?P<key>[^/]*)$', 'activate_account', name="activate"),
+
+    url(r'^accounts/disable_account_ajax$', 'disable_account_ajax', name="disable_account_ajax"),
+    url(r'^accounts/manage_user_standing', 'manage_user_standing', name='manage_user_standing'),
+
+    url(r'^change_setting$', 'change_setting', name='change_setting'),
+    url(r'^change_email_settings$', 'change_email_settings', name='change_email_settings'),
+)
+
+# enable automatic login
+if settings.FEATURES.get('AUTOMATIC_AUTH_FOR_TESTING'):
+    urlpatterns += (
+        url(r'^auto_auth$', 'auto_auth'),
+    )
+
+urlpatterns = patterns(*urlpatterns)
+
+# password reset mixes student.views with django views
```

Comment:
Ah, yes, that makes more sense. Ideally, I think you'd cluster all of the views that use the `student.views` prefix, and call `patterns` on them inline, and also group the `django.contrib.auth` views in a separate group with a prefix, rather than the current method which updates the `urlpatterns` variable multiple times.